common/lib/capa/capa/inputtypes.py

```diff
@@ -355,7 +358,7 @@ def get_html(self):
 
         context = self._get_render_context()
 
-        html = self.capa_system.render_template(self.template, context)
+        html = self.capa_system.render_template(self.template, context).strip()
```

Comment:
@muhammad-ammar Can we put a check that if `html` exist then do `strip` for safe side.